cmd/activator/request_log_test.go

```diff
@@ -114,6 +140,12 @@ func TestUpdateRequestLogFromConfigMap(t *testing.T) {
 	}
 }
 
+func getBoolPtr(val bool) *bool {
+	ptr := new(bool)
+	*ptr = val
+	return ptr
+}
+
```

Comment:
Maybe there is an `option` impl to use.

Reply:
There is `knative.dev/pkg/ptr` and then `ptr.Bool`.